qt/aqt/preferences.py

```diff
@@ -240,6 +252,20 @@ def sync_logout(self) -> None:
         self.mw.col.media.force_resync()
         self.update_login_status()
 
+    def ankihub_sync_login(self) -> None:
+        def on_success():
+            if self.mw.pm.ankihub_token():
+                self.update_login_status()
+
+        ankihub_login(self.mw, on_success, from_prefs_screen=True)
+
+    def ankihub_sync_logout(self) -> None:
+        def on_success():
+            self.mw.pm.set_ankihub_token(None)
+            self.update_login_status()
+
+        ankihub_logout(self.mw, on_success, self.mw.pm.ankihub_token())
```

Comment:
:pick: `ankihub_sync_login` doesn't set the credentials (they are set in `ankihub_login` instead) and `ankihub_logout` does set the credentials. By setting credentials I mean calling `mw.pm.set_ankihub_token` and `mw.pm.set_ankihub_username`. It would be better to make this consistent.